### PR TITLE
__version__   in std lib

### PR DESCRIPTION
**Pull request by Abdur-rahmaanJ**

### CODE REVIEW

**Comment by PythonCHB** [comment]
Another conflict -- also the README. I think if you fix the other one first, this may go away.

File: source/python_list/purpose_of___version___in_std_library.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+###############################################################
```
Comment (PythonCHB) [inline source/python_list/purpose_of___version___in_std_library.rst:1]: This is a pretty minor issue -- if you want to put in pages like this, we may want to create a bit more structure.

I see this project as mostly about capturing:

A) common issues that will come up again
B) collecting / summarizing large threads that are hard to find and read on the list archives.

Comment (Abdur-rahmaanJ) [reply source/python_list/purpose_of___version___in_std_library.rst:1]: @PythonCHB ah yes in priority, then earmark them then we come to those like this.

**Comment by Abdur-rahmaanJ** [comment]
@PythonCHB maybe add a todo with links

File: source/python_list/purpose_of___version___in_std_library.rst (hunk 1 above)
Comment (PythonCHB) [reply source/python_list/purpose_of___version___in_std_library.rst:1]: or use issues for that...

**Pull request merged**